roles/openshift_hosted_facts/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Set legacy hosted facts
+  openshift_facts:
+    role: hosted
+    openshift_env:
+      openshift_hosted_router_selector: "{{ openshift_router_selector | default(None) }}"
```

Comment:
openshift_env fact migrations are now occurring in the `oo_openshift_env` filter.